package.json

```diff
@@ -12,7 +12,7 @@
     "build": "run-s -l build:angular1 build:angular13",
     "build:angular1": "grunt build",
     "build:angular13": "ng build",
-    "serve:angular13": "export NODE_OPTIONS=--max_old_space_size=4096 && ng serve --host 0.0.0.0",
+    "serve:angular13": "set NODE_OPTIONS=--max_old_space_size=8000 && ng serve --host 0.0.0.0",
```

Comment:
**question:** I am curious of why this command is changed as part of this PR? 🤔

Reply:
> **question:** I am curious of why this command is changed as part of this PR? 🤔

I think it because he's using windows, and commit this file by accident.

Reply:
Cause I am using Windows, If I use "export", I cannot run doubt-fire locally, means I cannot hot update by any changes, so I need this command, this method is helped by Ray.

Reply:
> Cause I am using Windows, If I use "export", I cannot run doubt-fire locally, means I cannot hot update by any changes, so I need this command, this method is helped by Ray.

This kind of changes should not push to github, so it would be great that you don't use ` git add .` to add all files. You can add the files need to be push to github one by one using command `git add <filename>` or use VScode git control panel (stage change means git add).

Reply:
Would this be problematic for non-windows users? How about an alternative that could work in both systems? 🤔 

```js
"serve:angular13": "NODE_OPTIONS=--max_old_space_size=8000 ng serve --host 0.0.0.0",
```

Reply:
> Would this be problematic for non-windows users? How about an alternative that could work in both systems? 🤔
> 
> ```js
> "serve:angular13": "NODE_OPTIONS=--max_old_space_size=8000 ng serve --host 0.0.0.0",
> ```

Yes, this would be an error like:
![image](https://user-images.githubusercontent.com/33137074/168464033-448dcd04-748a-4034-a5d1-3711b1743ee5.png)

Reply:
It would also be nice to address this issue in a separate PR.